src/main/java/io/kuzzle/sdk/core/Kuzzle.java

```diff
@@ -1823,13 +1827,39 @@ private void  cleanQueue() {
     }
   }
 
+  private void  mergeOfflineQueueWithLoader() {
+    KuzzleQueue<KuzzleQueryObject> additionalOfflineQueue = this.offlineQueueLoader.load();
```

Comment:
Since it's impossible to know what kind of exception will be thrown, it is useless. The load method doesn't explicitly throws any exception, the developer will have to encapsulate his own code into try..catch